cmp/layout/TileFrame.ts

```diff
@@ -101,7 +101,8 @@ export const [TileFrame, tileFrame] = hoistCmp.withFactory({
             childrenArr.map((item, idx) => box({
                 style: localModel.getTileStyle(idx),
                 className: 'xh-tile-frame__tile',
-                item
+                item,
+                key: isObject(item) && 'key' in item ? item.key : undefined
             })):
```

Comment:
I think an explanation in the form of a comment is needed here, to explain why passing the item keys to the box is necessary.  What issues is this solving/avoiding?